xunit-autolabeler-v2/ast_parser/python/source_parsers/webapp2_router.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, List
+
+
+from ast_parser.python import drift_data_tuple
+
+
+def parse(nodes: List[Any]) -> List[Any]:
+    """Identify webapp2 route-handling snippets in Python
+       files and extract their language-agnostic data
+    Args:
+        nodes: A list of AST nodes obtained via the 'ast' package
+        class_name: the class name of the Python file being parsed
+    Returns:
+        A list of webapp2-handling methods (as AST nodes) with
+        added language-agnostic data (in the 'drift' attribute)
+    """
+
+    wsgi_configs = [node for node in nodes if
+                    hasattr(node, 'value') and
+                    hasattr(node.value, 'func') and
+                    hasattr(node.value.func, 'attr') and
+                    node.value.func.attr == 'WSGIApplication']
+
+    class_name_url_map = {}
+    for method in wsgi_configs:
+        method.drift = {}
+        arg = method.value.args[0]
```

Comment:
What if method.value.args list is empty? Should there be an emptiness check here and skip the method is no args to avoid a an index out of range error?

Reply:
Done. (Added a check.)